### PR TITLE
initial

### PR DESCRIPTION
**Pull request by SumirSeth**

### CODE REVIEW

File: domains/sumir.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Sumir Seth",
```
Comment (MaskDuck) [inline domains/sumir.json:3]: gh username pls?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/sumir.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "sumirseth",
+    "email": "iamsumirseth@gmail.com"
+  },
+  "record": {
+    "CNAME": "sumirseth.github.io"
```
Comment (MaskDuck) [inline domains/sumir.json:7]: 404?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/sumir.json (hunk 1 above)
Comment (SumirSeth) [reply domains/sumir.json]: cool now?

File: domains/sumir.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "sumirseth",
+    "email": "iamsumirseth@gmail.com"
+  },
+  "record": {
+    "CNAME": "sumirseth.github.io/sumir"
```
Comment (MaskDuck) [inline domains/sumir.json:7]: no cname at slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Pull request closed without merging**